frame/bounties/src/weights.rs

```diff
@@ -61,87 +61,91 @@ pub trait WeightInfo {
 /// Weights for pallet_bounties using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-	// Storage: Treasury BountyCount (r:1 w:1)
+	// Storage: Bounties BountyCount (r:1 w:1)
 	// Storage: System Account (r:1 w:1)
-	// Storage: Treasury BountyDescriptions (r:0 w:1)
-	// Storage: Treasury Bounties (r:0 w:1)
+	// Storage: Bounties BountyDescriptions (r:0 w:1)
+	// Storage: Bounties Bounties (r:0 w:1)
```

Comment:
do we need to worry about a storage migration here?

Reply:
Storage was already migrated in https://github.com/paritytech/polkadot/pull/3704 and #9566 . The change in output is just because the weights files are old